remo/profiles/tasks.py

```diff
@@ -83,6 +83,7 @@ def reset_rotm_nominees():
         nominees = UserProfile.objects.filter(is_rotm_nominee=True)
         for nominee in nominees:
             nominee.is_rotm_nominee = False
+            nominee.rotm_nominated_by = None
```

Comment:
Yap. I was also thinking of it. But implementing another db table for ROTM feature needs some code re-factor. This improvements should be done, but I think in separate PR/Bug. Can you please open a new bug and cc me there?

> This way we won't have to reset the nominations every month - we can simply create a new entry, and we can also have a modelform for the ROTM nominees.

Totally agree